example.nxc

```diff
@@ -6,18 +6,28 @@
 task main()
 {
     SetSensorNXTViSy(PORT);
-    NXTViSyInit(SIGNATURE, 10, 24 * 100);
+    long err = NXTViSyInit(SIGNATURE, 10, 24 * 100);
+    if (err != NO_ERR) {
+        NumOut(0, LCD_LINE4, err);
```

Comment:
@vargac There should probably be also some textual information on the LCD screen which would help trace the problem to this line.

Something like `TextOut(0, LCD_LINE3, "Error: NXTViSyInit")` perhaps?